backend/src/app/services/vector_db/base.py

```diff
@@ -58,9 +77,85 @@ async def ensure_collection_exists(self) -> None:
         """Ensure the collection exists in the vector database."""
         pass
 
-    @abstractmethod
+    async def get_embeddings(
```

Comment:
I have moved these methods to the base class because they're shared by both Qdrant and Milvus.

Reply:
Awesome! Thanks